Makefile

```diff
@@ -36,10 +36,10 @@ endif
 
 build:
 ifeq ($(OS),Windows_NT)
-	make genproto
```

Comment:
seems like a valid solution to me if it works. the only thing I don't like is that we have duplicate `generate.sh` for each dir. maybe we can use `go generate protoc` directly ? and leave the `.sh` file only for api where there is a special case

Reply:
from the docs - ` Go generate is never run automatically by go build, go get, go test, and so on. It must be run explicitly. `
that means it won't fix `go get` right ?

Reply:
since go get doesn't run make build

Reply:
- removed `.sh` where inline command makes sense
- I think it will fix `go get` simply because there is now go files in the directories 

I'm working with Anton on testing it

Reply:
@antonlerner @y0sher check it in my fork, it didn't work.
Currently, I can`t see another solution besides checking in the generated pb files